### PR TITLE
Release google-cloud-talent 0.1.0

### PR DESCRIPTION
**Pull request by TheRoyalTnetennba**
* Initial release

This pull request was generated using releasetool.

### CODE REVIEW

**Comment by TheRoyalTnetennba** [comment]
It just needs a top-level README entry

**Comment by TheRoyalTnetennba** [comment]
README product url is incorrect

File: google-cloud-talent/CHANGELOG.md [hunk 1]
```diff
@@ -1 +1,5 @@
 # Release History
```
Comment (quartzmo) [inline google-cloud-talent/CHANGELOG.md]: It seems that the generated `CHANGELOG.md` needs an extra blank line in order to play well with releasetool. Can you manually add one here for now?

Comment (TheRoyalTnetennba) [reply google-cloud-talent/CHANGELOG.md]: just saw this. I'll fix it

**Pull request closed without merging**